### ISSUE TITLE
Cannot achieve the same results with InceptionV3 model on the validation set of ILSVRC12

### ISSUE THREAD

**Issue by Openning07** [tensorflow/models#2417]
Recently, I've working on my research based on the InceptionV3 model, with Tensorflow and Tensorpack.
It seems that I've managed to load all the necessary tensor variables in the checkpoint file (LOG: **The following variables are in the checkpoint, but not found in the graph: global_step:0**).
Besides, I change my data pipeline (in the test/inference stage) just the same as it is in https://github.com/tensorflow/models/blob/master/slim/preprocessing/inception_preprocessing.py.

After that, I get classification results on the validation set of ILSVRC12: 70.0 (top1) | 90.8 (top5). But the stated results are 78.0 (top1) | 93.9 (top5).

Have you ever met the same or similar problems?

**Comment by poxvoculi** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
